ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/LastRunStatusState.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactElement } from 'react';
+import { ReportStatus } from 'types/report.proto';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+
+type LastRunStatusStateProps = {
```

Comment:
Nit: I find it helps to always export the component props as a lot of times we need access to them for consuming components. This tends to come up a lot when there is a moderate amount of prop drilling (2-3+ levels) and we want to define the prop of a parent component to be the same type of the prop of a child component without duplicating the type at each level.